test/committee_test.exs

```diff
@@ -49,14 +49,44 @@ defmodule CommitteeTest do
       end)
     end
 
-    test "existing committee hooks", context do
+    test "outdated existing committee hooks should update to new version", context do
```

Comment:
Should we also update files that were generated by committee, that don't contain the `version` in it? Or as we didn't published the changes, we are safe assuming that everyone will have the proper identifier and version in place.

Reply:
Yup, that was exactly where I was coming from - right now no one would have a githook file with identifier but without versions, so it's safe to push this out :)